### ISSUE TITLE
Overloading function call is not working

### ISSUE THREAD

**Issue by yangbinnnn** [ethereum/remix-ide#1460]
My contract like this, overloading function `foo(uint256 n)` is not working(no value return). If the contract just hava `foo(uint256 n)`, it works.

```
contract OverloadContract {
// working
function foo(uint256 n, uint256 m) public pure returns (uint256) {
    return n + m;
}

// not working ?
function foo(uint256 n) public pure returns (uint256) {
    return n;
}

// working
function foo() public pure returns (uint256) {
    return 1;
}

// working
function foo(uint256 i, uint256 n, uint256 m) public pure returns (uint256) {
    return i + n + m;
}
}
```

**Comment by yann300** [comment]
That issue is fixed by https://github.com/ethereum/remix/pull/864
This is going to be shipped with Remix 0.7.

**Comment by yann300** [comment]
this is deployed to remix live
